### PR TITLE
Update UG page

### PR DESCRIPTION
**Pull request by bryansendeavour**
Contributes to #24.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -51,98 +51,141 @@ Paidlancers is a **desktop app for keeping track of your freelancing events, opt
...
-Adding a contact : `newcontact`
+### Adding a contact : `newcontact`
+
 Adding a contact to contacts
-Format: newcontact NAME NUMBER
-Creates a new contact with specified NAME and NUMBER
-Example:
-newcontact Deborah Tan /num 91234567
 
+**Format** : `newcontact n/NAME p/NUMBER`
```
Comment (glozxi) [inline docs/UserGuide.md:58]: Minor issue of extra spacing before ":" here.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -51,98 +51,141 @@ Paidlancers is a **desktop app for keeping track of your freelancing events, opt
...
-The index must be a positive integer 1, 2, 3, …​
+
+**Format**: `newrate INDEX AMOUNT`
+
+- Adds a specified rate, `AMOUNT`, to an event at the specified `INDEX`.
+
+  - The `INDEX` refers to the index number in the displayed events list.
+  - The `INDEX` must be a positive integer 1, 2, 3, …​
+
 Example:
```
Comment (glozxi) [inline docs/UserGuide.md:115]: Should "Example" be bold or normal?

**Review by glozxi — CHANGES_REQUESTED** [review]
Looks good overall, just some nits.

File: docs/UserGuide.md (hunk 1 above)
Comment (bryansendeavour) [reply docs/UserGuide.md]: 👍

File: docs/UserGuide.md (hunk 2 above)
Comment (bryansendeavour) [reply docs/UserGuide.md]: Made the changes! 👍

File: docs/UserGuide.md [hunk 3]
```diff
@@ -51,98 +51,141 @@ Paidlancers is a **desktop app for keeping track of your freelancing events, opt
...
+The event must not have time associated with it beforehand.
+
+
+### Saving all events and contacts: `save`
 
-Saving all events and contacts: save
 Saves all entries in Paidlancer.
-Format: save
+
+**Format**: `save`
```
Comment (glozxi) [inline docs/UserGuide.md:181]: I think we don't have to include the save and load commands because there is autosave.

File: docs/UserGuide.md [hunk 4]
```diff
@@ -51,98 +51,141 @@ Paidlancers is a **desktop app for keeping track of your freelancing events, opt
...
-list followed by delete 2 deletes the 2nd event in the event book.
+
+**Format**: `deleteevent INDEX`
+
+- Deletes the event at the specified INDEX
+  - The `INDEX` refers to the index number shown in the displayed event list.
+  - The `INDEX` must be a positive integer 1, 2, 3, …​
+  
+**Examples**:
+  list followed by delete 2 deletes the 2nd event in the event book.
```
Comment (glozxi) [inline docs/UserGuide.md:161]: Should this be a bullet point and the commands in code format?

File: docs/UserGuide.md [hunk 5]
```diff
@@ -51,98 +51,141 @@ Paidlancers is a **desktop app for keeping track of your freelancing events, opt
...
-Examples:
-list followed by delete 2 deletes the 2nd event in the event book.
+
+**Format**: `deleteevent INDEX`
+
+- Deletes the event at the specified INDEX
+  - The `INDEX` refers to the index number shown in the displayed event list.
+  - The `INDEX` must be a positive integer 1, 2, 3, …​
+  
+**Examples**:
```
Comment (glozxi) [inline docs/UserGuide.md:160]: Should this be a singular?

**Review by glozxi — CHANGES_REQUESTED** [review]
Looks good overall, but there are some nits that I didn't catch just now, sorry!

File: docs/UserGuide.md (hunk 3 above)
Comment (bryansendeavour) [reply docs/UserGuide.md]: 👍

**Review by glozxi — APPROVED** [review]
LGTM.

**Pull request merged**